api/main.py

```
@@ -9,6 +9,38 @@
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+gemini_api_key = os.getenv("GEMINI_API_KEY")
+if not gemini_api_key:
+    raise ValueError("GEMINI_API_KEY not found in .env file")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)